core/services/ocr2/plugins/ocr2keeper/evm21/streams_lookup.go

```diff
@@ -378,7 +382,7 @@ func (r *EvmRegistry) singleFeedRequest(ctx context.Context, ch chan<- MercuryDa
 				return err1
 			}
 
-			if resp.StatusCode == http.StatusNotFound || resp.StatusCode == http.StatusInternalServerError {
+			if resp.StatusCode == http.StatusNotFound || resp.StatusCode == http.StatusInternalServerError || resp.StatusCode == http.StatusBadGateway {
```

Comment:
can you document this change also in the ticket and PR summary

Reply:
done